src/hera/workflow.py

```diff
@@ -83,6 +85,16 @@ def __init__(
         self.image_pull_secrets = image_pull_secrets
         self.workflow_template_ref = workflow_template_ref
 
+        if ttl_strategy:
```

Comment:
Maybe this should be set using `setattr`? 🤔 Thoughts? Similar to [line 107](https://github.com/argoproj-labs/hera-workflows/pull/127/files#diff-74e2ee6a97c31386ee31e5ff0f31cca730e28e83f429fb64b421bc14b9197db1L107).